### PR TITLE
#83 Reorganize schemas

### LINKED ISSUE #83: ADT 3.0 - Reorganize schemas

### PR DESCRIPTION
**Pull request by fredrikbjornoy**
closes #83

### CODE REVIEW

File: asyncapi/asyncapi.yml [hunk 1]
```diff
@@ -251,33 +250,31 @@ channels:
             retain: true
   pe/dpi_ack:
```
Comment (fredrikbjornoy) [inline asyncapi/asyncapi.yml]: @eivindga should we change this to `pe/dpi/ack`?

File: asyncapi/asyncapi.yml [hunk 2]
```diff
@@ -251,33 +250,31 @@ channels:
...
-          { $ref: examples/dpi-acknowledge.json }
-        ]
+          $ref: json-schemas/pe/dpi_ack/dpi-acknowledge.json
+        examples:
+          - $ref: json-schemas/pe/dpi_ack/dpi-acknowledge.example.json
         bindings:
           mqtt:
             qos: 1
             retain: false
   pe/dpi_diagnostics:
```
Comment (fredrikbjornoy) [inline asyncapi/asyncapi.yml]: @eivindga should we change this to `pe/dpi/diagnostics`?

File: asyncapi/asyncapi.yml [hunk 3]
```diff
@@ -319,202 +316,186 @@ channels:
...
-          { $ref: examples/dpi-key_stops.json }
-        ]
+          $ref: json-schemas/pe/dpi/key_stops/dpi-key_stops.json
+        examples:
+          - $ref: json-schemas/pe/dpi/key_stops/dpi-key_stops.example.json
         bindings:
           mqtt:
             qos: 1
             retain: true
   pe/dpi_command:
```
Comment (fredrikbjornoy) [inline asyncapi/asyncapi.yml]: @eivindga should we change this to `pe/dpi/command`?

File: asyncapi/asyncapi.yml (hunk 1 above)
Comment (eivindga) [reply asyncapi/asyncapi.yml]: yes

File: asyncapi/asyncapi.yml (hunk 2 above)
Comment (eivindga) [reply asyncapi/asyncapi.yml]: yes

File: asyncapi/asyncapi.yml (hunk 3 above)
Comment (eivindga) [reply asyncapi/asyncapi.yml]: yes

**Review by eivindga — APPROVED** [review]

**Pull request merged**